pkg/operator/nodecadaemon.go

```diff
@@ -37,12 +39,14 @@ type NodeCADaemonController struct {
 }
 
 func NewNodeCADaemonController(
+	kubeClient *kubeclient.Clientset,
```

Comment:
can we pass an eventrecorder instead?  like https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/pkg/operator/starter.go#L114

Reply:
yep